connector/connect/server/pom.xml

```diff
@@ -28,7 +28,7 @@
 
   <artifactId>spark-connect_2.12</artifactId>
   <packaging>jar</packaging>
-  <name>Spark Project Connect</name>
+  <name>Spark Project Connect Server</name>
```

Comment:
We should also update the actual name of the project. We can do that in a follow-up.